SimTKmath/tests/CMAESTest.cpp

```diff
@@ -514,7 +515,9 @@ void testMultithreading() {
     Optimizer opt(sys, SimTK::CMAES);
     opt.setConvergenceTolerance(1e-12);
     opt.setMaxIterations(5000);
-    opt.setAdvancedRealOption("init_stepsize", 0.3);
+    Vector initStepSize(N, 0.3);// test vector option
+    opt.setAdvancedVectorOption("init_stepsize", initStepSize);
+    //opt.setAdvancedRealOption("init_stepsize", 0.3);// to test exception if both are set
```

Comment:
Should remove commented-out line, or better would be to leave in a test somewhere to verify that the "Real" option is working correctly.